### PR TITLE
JI-4534 switch from coverMedium to coverMedia

### PR DESCRIPTION
**Pull request by Gwenillia**

### CODE REVIEW

File: upgrades.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ H5PUpgrades['H5P.InteractiveBook'] = (function () {
           };
 
           if (bookCover.coverAltText || bookCover.coverImage) {
-            bookCover.coverMedia = convertToImageParams(bookCover.coverImage, bookCover.coverAltText);
+            bookCover.coverMedium = convertToImageParams(bookCover.coverImage, bookCover.coverAltText);
```
Comment (Gwenillia) [inline upgrades.js:78]: Why 🥺 If we want to change to cover media we should also change variable names I changed coverMedium to coverMedia everywhere 😓

Comment (ravimajithia) [reply upgrades.js:78]: The reason is this whole code is for the 1.6 version and we are already in the 1.7 version so this code will not execute at all.

File: semantics.json [hunk 2]
```diff
@@ -61,10 +61,10 @@
         }
       },
       {
-        "name": "coverMedium",
+        "name": "coverMedia",
```
Comment (Languafe) [inline semantics.json:64]: We don't want make a change like this unless there is a good reason for it. Changing the actual field name will require code changes and an upgrade script, and it adds zero value for users. Changing the label and description is all that is needed.

```suggestion
        "name": "coverMedium",
```

**Comment by Languafe** [comment]
I will close this pull request as it's easier to solve this from scratch instead of dealing with merging language files and reverting the changes.

**Pull request closed without merging**